caluma/form/serializers.py

```diff
@@ -534,12 +534,16 @@ class SaveDocumentTableAnswerSerializer(SaveAnswerSerializer):
     )
 
     def _get_document_tree(self, document_id):
-        answers = models.AnswerDocument.objects.filter(document_id=document_id).values(
-            "answer"
+        """Get all child documents (table rows) of a given document, and the document itself."""
+
+        answers = models.Answer.objects.filter(document_id=document_id)
+        child_documents = (
+            answers.filter(documents__isnull=False)
```

Comment:
I have a feeling isnull on a m2m is always false... with change above this would not really be needed though.